setup.py

```diff
@@ -67,7 +76,7 @@
             'pyobjc-core>=3.0.3',
             'pyobjc-framework-Cocoa>=3.0.3',
             'pyobjc-framework-Quartz>=3.0.3',
-            'wxPython>=2.8',
+            'wxPython>=3.0',
```

Comment:
Yep... is it okay to get this update on all platforms? Are you also using 3+ on Linux?